src/renderer/services/bitcoin/balances.ts

```diff
@@ -9,15 +9,23 @@ import { client$ } from './common'
  * e.g. @see src/renderer/services/wallet/balances.ts:getChainBalance$
  */
 const { get$: reloadBalances$, set: setReloadBalances } = observableState<boolean>(false)
+const { get$: reloadLedgerBalances$, set: setReloadLedgerBalances } = observableState<boolean>(false)
 
-const resetReloadBalances = () => {
-  setReloadBalances(false)
+const resetReloadBalances = (walletType: WalletType) => {
+  if (walletType === 'keystore') {
```

Comment:
It would be great to define enum for `WalletType`, cuz we might need more wallets to integrate.